x/auth/vesting/client/cli/tx.go

```diff
@@ -79,3 +83,90 @@ timestamp.`,
 
 	return cmd
 }
+
+type VestingData struct {
+	StartTime int64         `json:"start_time"`
+	Periods   []InputPeriod `json:"periods"`
+}
```

Comment:
instead of having a micro-format here for input, why don't we just decide that the .json file contains the proto-JSON representation of `PeriodicVestingAccount`:

```
{
  "base_vesting_account": {"base_account":{...}, "original_vesting":[]},
  "start_time": 123,
  "vesting_periods": [{"length": 123, "coins": [{"denom": test, "amount": 12}]}]
}

Reply:
I think the microformat is much more useable than leaking things like base vesting account to the user interface.